src/data/showcases.js

```diff
@@ -1156,6 +1156,15 @@ export const Showcases = [
     source: null,
     tags: ["governance"],
   },
+  {
+    title: "Moments",
+    description:
+      "A never-ending timeline where creators, share, own and govern their decentralized legacies.",
+    preview: require("./showcase/moments.png"),
+    website: "https://adamoments.io/",
+    source: null,
+    tags: ["analytics","marketplace","minting","nftsupport"],
```

Comment:
```suggestion
    tags: ["marketplace", "minting", "nftsupport"],
```
The `analytics` tag is definitely not appropriate (since no general analytics are provided here), and in any case there needs to be a space after each of these commas.